guides/17_Migration_to_3_0.md

```diff
@@ -162,6 +162,64 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## AudioVideo events
+
+The following `AudioVideo` events have been removed in v3.x, these events redirect data from `metricDidReceive`.
```

Comment:
From earlier reviews learnings, suggest using active voice (suggestion below):

We have removed below events in v3:
- videoSendHealthDidChange
- videoSendBandwidthDidChange
- videoReceiveBandwidthDidChange

`metricsDidReceive` observer now provides the metrics that were earlier included with above removed events.

Other Notes:
- Using v3 is fine as the migration now uses "v3" over "v3.x".
- We have followed "// Before" and "// After", dropping "Before in 2.x" and "After in 3.x". So suggest to change accordingly.

Reply:
Gotcha, updated. Also updated at a couple other inconsistent spots.